programs/ed25519/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "solana-ed25519-program"
+description = "Solana Ed25519 program"
+version = "1.8.0"
+homepage = "https://solana.com/"
+documentation = "https://docs.rs/solana-ed25519-program"
+repository = "https://github.com/solana-labs/solana"
+authors = ["Solana Maintainers <maintainers@solana.foundation>"]
+license = "Apache-2.0"
+edition = "2018"
+
+[dependencies]
+solana-sdk = { path = "../../sdk", version = "=1.8.0" }
+
+[dev-dependencies]
+bincode = "1.3.3"
```

Comment:
Would borsh be more efficient?

Reply:
I've left it as for now while we decided on bytemuck (I like the name btw)